test_extras/test_chemprop/test_models.py

```diff
@@ -28,10 +28,10 @@
 # pylint: disable=relative-beyond-top-level
 from test_extras.test_chemprop.chemprop_test_utils.compare_models import compare_params
 from test_extras.test_chemprop.chemprop_test_utils.constant_vars import (
-    NO_IDENTITY_CHECK,
```

Comment:
@JochenSiegWork I think we need to check our isort settings

Reply:
but it went through? I just ran 'isort . --profile black' with latest isort...

Reply:
Yes, the issue is, that apparently in the last PR the imports were not in the correct order, but the pipeline still passed.

Reply:
Created [issue](https://github.com/basf/MolPipeline/issues/87)